test/Tests.hs

```diff
@@ -37,44 +39,67 @@ main = do
             , fixtureGoldenTest "copy-fields-some"
             , fixtureGoldenTest "copy-fields-none"
             ]
+        , testGroup "github-specific"
```

Comment:
unnecessary complication.

Please, do the least required stuff.

Reply:
Do you mean, remove the extra `testGroup "github-specific"` layer, but keep `fixtureWeederGoldenTest`?  I've removed the extra group layer now.

Reply:
Or if you mean that `fixtureWeederGoldenTest` is too complicated, could you help give an idea of what a minimal test would look like?